app/components/views/HomePage/Page.js

```diff
@@ -35,9 +36,9 @@ const HomePage = ({
           <div>
             <Balance amount={spendableTotalBalance} />
             <div className="home-rescan-button-area" data-html={true} data-tip="Rescanning may help resolve some balance errors.<br><br><b>Note:</b> This scans the entire blockchain for transactions,<br>but does not re-download it.">
-              <KeyBlueButton onClick={() => rescanAttempt(0)}>Rescan Blockchain</KeyBlueButton>
+              <KeyBlueButton disabled={rescanRequest} onClick={() => rescanAttempt(0)}>Rescan Blockchain</KeyBlueButton>
             </div>
-            <ReactToolTip place="left" type="info" effect="solid"/>
+            <ReactToolTip disable={rescanRequest ? true : false} place="left" type="info" effect="solid"/>
```

Comment:
This can be simplified to `disable={rescanRequest}`.

Reply:
It can't because react tooltip requires a boolean and rescanRequest is an object.

Reply:
Ah, I see... what about `!!rescanRequest`?

Reply:
Well... I personally dislike double negations but if more people think it's better, then sure I can change it.

Reply:
Nah, I'm just nitpicking.